data/data_parser.py

```diff
@@ -0,0 +1,42 @@
+import os
+import json
+import datasets
+from transformers import ElectraTokenizer
+import argparse
+
+
+def get_txt_files(directory):
+    txt_files = [os.path.join(directory, f) for f in os.listdir(directory) if f.endswith('.txt')]
+    return txt_files
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--BASE_PATH", default="/workspace/intent-classification-analysis/data/raw")
+    parser.add_argument("--id_info_path", default="/workspace/intent-classification-analysis/data/id.json")
```

Comment:
```suggestion
    from os.path import abspath, dirname, join
    
    ROOT_DIR = dirname(abspath(__file__))
    DATA_DIR = join(ROOT_DIR, "data")

    parser.add_argument("--BASE_PATH", default=join(DATA_DIR, "raw"))
    parser.add_argument("--id_info_path", default=default=join(DATA_DIR, "id.json"))
```

1. 개발환경에 따라 절대경로가 변할 수 있기 때문에 project root directory를 기준으로 경로를 잡는 것이 나을 것 같습니다.
2. `save_dir`도 마찬가지